components/Trade/TradingPanel/Account/index.tsx

```diff
@@ -11,10 +11,8 @@ import { LIMIT } from '@libs/types/OrderTypes';
 import TooltipSelector from '@components/Tooltips/TooltipSelector';
 import { UserBalance } from '@libs/types';
 import ConnectOverlay from '@components/Overlay/ConnectOverlay';
-import { CalculatorStore1 } from '@context/CalculatorContext1';
-import { CalculatorStore2 } from '@context/CalculatorContext2';
-import CalculatorModal1 from './CalculatorModal1';
-import CalculatorModal2 from '@components/Trade/TradingPanel/Account/CalculatorModal2';
+import { CalculatorStore2 } from '@context/CalculatorContext';
```

Comment:
should this be renamed to CalculatorStore??

Reply:
done. thanks